manifests-maistra/2.3.0/maistraoperator.v2.3.0.clusterserviceversion.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
       The Maistra Operator enables you to install, configure, and manage an instance of Maistra service mesh. Maistra is based on the open source Istio project.
 
     containerImage: quay.io/maistra/istio-ubi8-operator:2.3.0
-    createdAt: 2022-09-26T09:28:03MDT
+    createdAt: 2022-09-29T11:23:46EDT
```

Comment:
We usually avoid updating `createdAt`, but I am not sure if it is a problem before we release a particular version...?

Reply:
That's in the second commit. There are new changes generated from the `make gen` so I think I need to update the manifest date. If that's not good, I can checkout the original values

Reply:
Yes, but manifest did not change itself. @jwendell could you confirm?